tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "1.6.2",
+  "compilerOptions": {
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "target": "es5",
+    "module": "commonjs",
+    "sourceMap": true
+  },
+  "files": [
+    "src/app/bootstrap.ts",
+    "src/typings/tsd.d.ts"
+  ]
+}
```

Comment:
if we move this to the one i have in the src folder, we dont have to deal with specifying every file. that is not a fun experience nor how we should go as the app grows. the idea of putting tsconfig in the src folder means now every file under here is fair game. no need for excludes and includes, which are very easy to screw up.